### ISSUE TITLE
Process Frame return false

### ISSUE THREAD

**Issue by BingBingAR** [chrdiller/KinectFusionLib#8]
Hello.
I have built this project with visual studio 2015, cuda8.0 and opencv3.4.
Problem is : It shows "Frame could not be processed".
I tracked the code, found that in estimate_step() function at pose_estimation.cu, on line 208, all values from host_data ptr returns 0.
So line 54 of pose_estimation.cpp always gets in false.
Could you please give me any instruction?
Best Regards.

**Comment by JonathanAMichaels** [comment]
I'm having the same problem on Ubuntu 16.04 with cuda 8.0 and opencv 3.4.3 (compiled with cuda support). I've confirmed that data is streaming properly on both the realsense and kinect (I added a KinectCamera implementation). I'll continue trying to debug.

**Comment by JonathanAMichaels** [comment]
@BingBingAR Turns out I entered the wrong GPU Arch code in the cmake file (default is 5.2). Can you confirm that the code you entered matches your GPU? 
 https://en.wikipedia.org/wiki/CUDA#GPUs_supported

**Comment by BingBingAR** [comment]
Hi JonathanAMichaels.
Have you run the code in your machine?

**Comment by JonathanAMichaels** [comment]
Yes, the code runs properly on my machine. Make sure your GPU Arch code is correct, and make sure you've compiled OpenCV with CUDA support, which means installing both opencv and the opencv_contrib modules.

**Comment by BingBingAR** [comment]
Hi JonathanAMichaels. Thanks for  you reply.
I am running the code on windows10/GTX 1080Ti, but cuda functions return error code on every steps.
Could you please try on windows?
Thanks.

**Comment by ElephantGit** [comment]
> Yes, the code runs properly on my machine. Make sure your GPU Arch code is correct, and make sure you've compiled OpenCV with CUDA support, which means installing both opencv and the opencv_contrib modules.

Hi, JonathanAMichaels.
I have this process frame return false problem, too. And I run the solution on Ubuntu 18.04, CUDA9.1, GeForce 940MX, and OpenCV3.4.1. I set my GPU Arch code as: -gencode arch=compute_50,code=sm_50. I also compile OpenCV with CUDA, but it still return false. 
Could you please tell me if there is any wrong config?

**Comment by ElephantGit** [comment]
> Hello JonathanAMichaels.
> I have run the solution on Ubuntu 16.0.4 and I can see it is working.
> The problem is I am using intel d435 camera and texture is not mapping correctly.
> Does this solution make good texture mapping on your side?

Hi, BingBingAR.
I have the process frame return false problem, too. Could you please tell me how do you solve this problem?
